src/test/custom_loader_test.cpp

```diff
@@ -760,7 +760,7 @@ TEST_F(TestCustomLoader, CustomLoaderWithMissingModelFiles) {
         {{DUMMY_MODEL_INPUT_NAME,
             std::tuple<ovms::shape_t, tensorflow::DataType>{{1, 10}, tensorflow::DataType::DT_FLOAT}}});
     tensorflow::serving::PredictResponse response;
-    ASSERT_EQ(performInferenceWithRequest(request, response), ovms::StatusCode::MODEL_NAME_MISSING);
+    ASSERT_EQ(performInferenceWithRequest(request, response), ovms::StatusCode::MODEL_VERSION_MISSING);

```

Comment:
Just a question -> why the retCode differs here? I don't understand from where it comes from?

Reply:
This is because we have moved this line:
`auto model = getModelIfExistCreateElse(config.getName());`
  
By doing this, we create new model instance even if it errors to load. In such case, we have new model entry but with no versions. In predict, we return MODEL_NAME_MISSING if model is missing in model map, but MODEL_VERSION_MISSING if model is present but no version is available in instances map.